### ISSUE TITLE
[PATCH] spelling fix

### ISSUE THREAD

**Issue by gregoa** [ido50/Tenjin#4]
In Debian we are currently applying the following patch to Tenjin.
We thought you might be interested in it too.

```
Description: spelling fix
Origin: vendor
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2016-05-15
```

The patch is tracked in our Git repository at
https://anonscm.debian.org/cgit/pkg-perl/packages/libtenjin-perl.git/plain/debian/patches/spelling.patch

Thanks for considering,
  gregor herrmann,
  Debian Perl Group

**Comment by ido50** [comment]
Fixed in 1.0.1, thanks.

**Comment by gregoa** [comment]
On Tue, 17 May 2016 12:54:31 -0700, Ido Perlmuter wrote:

> Fixed in 1.0.1, thanks.

Great, thank you.

Cheers,
gregor

## 

 .''`.  Homepage https://info.comodo.priv.at/ - OpenPGP key 0xBB3A68018649AA06
 : :' : Debian GNU/Linux user, admin, and developer -  https://www.debian.org/
`. `'  Member of VIBE!AT & SPI, fellow of the Free Software Foundation Europe
`\-   NP: Bruce Springsteen: Glory Days
